fmt.pc

```diff
@@ -0,0 +1,11 @@
+prefix=@out@
```

Comment:
I suppose upstream probably also wants to see some sort of build system integration into cmake using `configure_file`:

https://dailycommit.blogspot.com/2016/10/how-to-generate-pkg-config-file-with.html